test/app/components/market/submit/parameter-input.component.spec.ts

```diff
@@ -175,6 +240,66 @@ describe("ParameterInputComponent", () => {
             expect(inputEl.nativeElement.value).toBe(String(updatedInput));
         });
 
+        it("should validate minimum value constraint", () => {
+            const input = 5;
+            testComponent.param = new NcjParameterWrapper("frameEnd", {
```

Comment:
for those test add the param restriction in the beforeEach() so you have less duplicate and just include both restriction at the same time.

Reply:
done